### DISCUSSION TITLE
Debugging enhancement for achievement editor: Individual processing ☑ per requirement

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAIntegration#773] [Ideas]
As an achievement developer anything that can save me time and makes developing easier is appreciated. Here's a concept which was first suggested by **Zaphneth** in the Discord Devs channel:

The purpose of this enhancement would be to improve the debugging and testing process of crafting achievements, and it's anticipated it would simplify this process.

### Mockup:
![image](https://user-images.githubusercontent.com/32706333/48309333-e6eaae80-e534-11e8-9b41-aa366c611b05.png)

In the achievement editor a checkbox per requirement would be added to the left of ID column. By default all boxes would be checked. A checked box would mean processing is active for that requirement; unchecked, processing disabled.

There would also be a box on top of the others, that if checked/unchecked, would change the checked state of all the requirements below.

If any of these boxes are tampered with, achievement processing would be disabled.

### Current: (for comparison)
![image](https://user-images.githubusercontent.com/32706333/48309300-42686c80-e534-11e8-9f46-05d67649a177.png)

**Comment by Keltron3030** [comment]
It's a good idea.  This would dovetail nicely with the concept of conditional formatting, where a condition line's background color would change green when true and red when false.  It could then be white when deactivated for example.
